sources/packages/backend/apps/queue-consumers/src/processors/schedulers/base-scheduler.ts

```diff
@@ -47,9 +49,23 @@ export abstract class BaseScheduler<T> implements OnApplicationBootstrap {
    * any old cron job delete it and add the new job to the queue.
    */
   async onApplicationBootstrap(): Promise<void> {
+    const isFulltimeAllowed = this.configService.isFulltimeAllowed;
     await this.deleteOldRepeatableJobs();
     // Add the cron to the queue.
     await this.schedulerQueue.add(await this.payload());
+    // Allow Fulltime Schedulers only if isFulltimeAllowed is true
```

Comment:
`deleteOldRepeatableJobs` and `await this.schedulerQueue.add(await this.payload());` will not be required if the queue needs to be obliterate

![image](https://github.com/bcgov/SIMS/assets/54600590/3ffa6e7c-5c4c-47de-b0df-cc7af56c78b6)